### PR TITLE
Add commit hash to finding location.

### PR DESCRIPTION
**Pull request by martinmr**
Part of PLAT-1576

### CODE REVIEW

File: src/types/scanText.ts [hunk 1]
```diff
@@ -26,10 +26,12 @@ export namespace ScanText {
     location: {
       byteRange: FindingRange
       codepointRange: FindingRange
+      commitHash: string
     };
     redactedLocation?: {
       byteRange: FindingRange
       codepointRange: FindingRange
+      commitHash: string,
```
Comment (evanfuller) [inline src/types/scanText.ts:34]:
```suggestion
      commitHash: string
```

File: src/tests/mocks.ts [hunk 2]
```diff
@@ -49,7 +49,8 @@ export const scanTextResponse = {
           "codepointRange": {
             "start": 25,
             "end": 41
-          }
+          },
+          "commitHash": "",
```
Comment (evanfuller) [inline src/tests/mocks.ts:53]:
```suggestion
          "commitHash": ""
```

**Review by evanfuller — APPROVED** [review]

File: src/tests/mocks.ts (hunk 2 above)
Comment (martinmr) [reply src/tests/mocks.ts]: Done

File: src/types/scanText.ts (hunk 1 above)
Comment (martinmr) [reply src/types/scanText.ts]: Done.

**Pull request merged**